lib/src/cgcloud/lib/ec2.py

```diff
@@ -124,7 +124,7 @@ class EC2VolumeHelper( object ):
     A helper for creating, looking up and attaching an EBS volume in EC2
     """
 
-    def __init__( self, ec2, name, size, availability_zone ):
+    def __init__( self, ec2, name, size, availability_zone, volume_type="standard" ):
         """
```

Comment:
I left "standard" as the default, but if gp2 is better all-around then we may want to change this